.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://npm.lwcjs.org
```

Comment:
There's a dependency tests have that's only available here at this registry. The dependency is:
lwc-wire-service-sfdc-mocks